os/android/gralloc1bufferhandler.cpp

```diff
@@ -122,6 +123,9 @@ bool Gralloc1BufferHandler::Init() {
   set_modifier_ = reinterpret_cast<GRALLOC1_PFN_SET_MODIFIER>(
       gralloc1_dvc->getFunction(gralloc1_dvc, GRALLOC1_FUNCTION_SET_MODIFIER));
 #endif
+
+  get_interlace_ = reinterpret_cast<GRALLOC1_PFN_GET_INTERLACE>(
```

Comment:
Encapsulate with "if (handle->meta_data_.usage_ == hwcomposer::kLayerVideo)"

Reply:
Disregard comment if there is no simple way to check if this is pertinent to video.